sycl/test/basic_tests/stream/blocking_pipes_and_stream.cpp

```diff
@@ -0,0 +1,83 @@
+// REQUIRES: accelerator
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out %ACC_CHECK_PLACEHOLDER
+
+// Test that usage of blocking pipes and stream object in the same kernel
+// doesn't cause program hang.
+
+// CHECK: 0
+// CHECK-NEXT: 1
+// CHECK-NEXT: 2
+// CHECK-NEXT: 3
+// CHECK-NEXT: 4
+// CHECK-NEXT: 5
+// CHECK-NEXT: 6
+// CHECK-NEXT: 7
+// CHECK-NEXT: 8
+// CHECK-NEXT: 9
+// CHECK-NEXT: 0
+// CHECK-NEXT: 1
+// CHECK-NEXT: 2
+// CHECK-NEXT: 3
+// CHECK-NEXT: 4
+// CHECK-NEXT: 5
+// CHECK-NEXT: 6
+// CHECK-NEXT: 7
+// CHECK-NEXT: 8
+// CHECK-NEXT: 9
+
+#include <CL/sycl.hpp>
```

Comment:
Just curious about why today this is picking the tested SYCL implementation and not the already installed SYCL implementation on the machine?
Are there specific LIT implementation or it assumes there is no SYCL implementation installed?

Reply:
Good question. I think you are right and using <> intead of "" could lead to some problems. But if we are going to fix this then it should be a bulk change in the whole project.